aplt/runner.py

```diff
@@ -586,10 +593,13 @@ def run_scenario(args=None, run=True):
     scenario_args = []
     scenario_kw = {}
     if arguments.scenario_args:
-        scenario_args, scenario_kw = group_kw_args(arguments.scenario_args)
+        scenario_args, scenario_kw = group_kw_args(*arguments.scenario_args)
```

Comment:
I think the other group_kw_args call for aplt_testplan (in parse_testplan) needs the same * fix